src/cpp/r/R/Tools.R

```diff
@@ -414,6 +414,13 @@ assign(envir = .rs.Env, ".rs.getVar", function(name)
       unlockBinding(name, packageEnv)
       assign(name, new, packageName)
       lockBinding(name, packageEnv)
+
+      if (namespace && package %in% loadedNamespaces()) {
```

Comment:
@kevinushey Thanks, from our slack chat, it does seem to be the case that `package %in% loadedNamespaces()` could be removed, in this case, since we are checking the `utils` package. However, to leave `registerHook` generic enough that others could use this functionality, we would still want to check the package is loaded since other packages might not be loaded by default, correct?

I don't fully understand the side effects from calling `asNamespace()`, in its documentation it mentioned that: `Internal namespace support functions. Not intended to be called directly, and only visible because of the special nature of the base namespace.`; however, this is used throughout the codebase.

I'm inclined to keep this as it is, but let me know if I might be missing something here.